manual/development-process/access-an-environment-db.md

```diff
@@ -0,0 +1,101 @@
+#Accessing an environment's database
+
+Connecting to a database must be done through an AWS Bastion host, below are the steps detailing how to do this
```

Comment:
Seems like a link to the other page for creating the bastion would be useful here.

Reply:
Yep, makes sense, I've added in a link